tools/cockpit.spec

```diff
@@ -1,10 +1,20 @@
-# Globals that might be defined elsewhere
```

Comment:
this comment is now in the wrong place, it should stay close to "* gitcommit xxx" etc below.